src/exportApis.ts

```diff
@@ -0,0 +1,33 @@
+export {
+  DepositsApiGetDepositRequest,
+  DepositsApiListDepositsRequest,
+  AssetsApiGetAssetRequest,
+  AssetsApiListAssetsRequest,
+  CreateCollectionRequest,
+  CollectionsApiGetCollectionRequest,
+  CollectionsApiListCollectionFiltersRequest,
+  CollectionsApiListCollectionsRequest,
+  UpdateCollectionRequest,
+  AddMetadataSchemaToCollectionRequest,
+  MetadataApiGetMetadataSchemaRequest,
+  MetadataSchemaRequest,
+  CreateProjectRequest,
+  BalancesApiGetBalanceRequest,
+  BalancesApiListBalancesRequest,
+  MintsApiGetMintRequest,
+  MintsApiListMintsRequest,
+  WithdrawalsApiListWithdrawalsRequest,
+  WithdrawalsApiGetWithdrawalRequest,
+  OrdersApiGetOrderRequest,
+  OrdersApiListOrdersRequest,
+  GetSignableCancelOrderRequest,
+  TradesApiGetTradeRequest,
+  TradesApiListTradesRequest,
+  GetSignableTradeRequest,
+  TokensApiGetTokenRequest,
+  TokensApiListTokensRequest,
+  TransfersApiGetTransferRequest,
+  TransfersApiListTransfersRequest,
+} from './api';
+
```

Comment:
How about configuration? https://github.com/immutable/imx-core-sdk/blob/v1.0.0-beta.1/src/api/index.ts#L17

Reply:
Do we need to? We provide constructors e.g. `Config.SANDBOX`

Reply:
Not sure, but we're [exporting them still. ](https://github.com/immutable/imx-core-sdk/blob/v1.0.0-beta.1/src/api/configuration.ts#L26) Why are we exporting them if we aren't exposing them?

Reply:
Not needed as it's autogenerated and. Discussed offline.